components/src/index.css

```
@@ -1,68 +1,232 @@
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
```

Biome (1.9.4)

[error] Unexpected shorthand property font after font-size



(lint/suspicious/noShorthandPropertyOverrides)